Noblesse/Character/Character.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Noblesse\Character;
+
+require_once $_SERVER['DOCUMENT_ROOT'] . "Noblesse/start.php";
+
+use Noblesse\Character\Interfaces\CharacterInterface;
+
+use const Noblesse\Character\Helpers\BASE_HEALTH;
+class Character
+{
+    private $name;
+    private $charType;
+    private $weaponType;
+    private $health;
+    private $damage;
+    private $minDamage;
+    private $maxDamage;
+    private $modHumanType;
+    private $inventory = [];
+    
+    /**
+     * Characteristics are defined
+     * @param string $newName       Name for the character
+     * @param string $newCharType   Type of character
+     * @param string $newWeaponType Fixed and cannot be changed
+     * 
+     * @var   int    $health        Default 100 
+     */
+    public function __construct(
+        string $newName,
+        string $newCharType,
+        string $newWeaponType
+    ) {
+        $this->name         = $newName;
+        $this->charType     = $newCharType;
+        $this->weaponType   = $newWeaponType;
+        $this->health       = BASE_HEALTH;
+    }
+
+    public function setName(string $newName)
+    {
+        $this->name = $newName;
+    }
+
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    public function setModHumanType(bool $isModHuman = false, string $type = '')
+    {
+        if (! $isModHuman) $this->modHumanType = NULL;
+
+        $this->modHumanType = $type; 
+    }
+
+    public function getModHumanType()
+    {
+        return $this->modHumanType;
+    }
+
+    public function setDamage(int $min, int $max)
+    {
+        $this->minDamage = $min;
+        $this->maxDamage = $max;
+    }
+    /**
+     * @return int Random int
+     */
+    public function getDamage()
+    {
+        return $this->damage = rand($this->minDamage, $this->maxDamage);
+    }
+
+    /**
+     * @return array Minimum and Max damage of the character
+     */
+    public function getMinMaxDamage()
+    {
+        return [
+            'min' => $this->minDamage,
+            'max' => $this->maxDamage
+        ];
+    }
+
+    public function setCharType(string $newCharType)
```

Comment:
@KyleMerc you can add a description of your getter and setter. define params, return type and basic description.